ethereum/spec/src/main/java/tech/pegasys/teku/spec/cache/CapturingIndexedAttestationCache.java

```diff
@@ -14,23 +14,11 @@
 package tech.pegasys.teku.spec.cache;
 
 import java.util.Collection;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.function.Supplier;
-import tech.pegasys.teku.spec.datastructures.operations.Attestation;
 import tech.pegasys.teku.spec.datastructures.operations.IndexedAttestation;
 
-public class CapturingIndexedAttestationCache implements IndexedAttestationCache {
-  private final Map<Attestation, IndexedAttestation> indexedAttestations = new HashMap<>();
-
+public class CapturingIndexedAttestationCache extends SimpleIndexedAttestationCache {
```

Comment:
Do we need to have this type hierarchy or would just using `CapturingIndexedAttestationCache` work?  The simple version just omits a getter which doesn't seem like it would hurt to have present even if it's not used. Especially since the `IndexedAttestationCache` already omits the getter anyway so could just use the interface to restrict the available API.

Reply:
We don't really need it.  I thought I needed the simple cache, but then realized I didn't and just left it.  I can merge them back.